airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/component/TableOperationsFixtures.kt

```diff
@@ -717,12 +717,35 @@ object TableOperationsFixtures {
             namespaceMapper = NamespaceMapper(),
         )
 
+    fun createSimpleStream(
+        namespace: String,
+        name: String,
+        schema: ObjectType,
+        isDedup: Boolean,
+        primaryKey: String = "id",
+        cursor: String? = "updated_at",
```

Comment:
Why not just take an `importType`? Then we can avoid the boolean flags and just pass importType's in.